src/guide-start.md

```diff
@@ -0,0 +1,68 @@
+# Getting Started
+
+Lets kick things off with an example ([playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=4cad6600b21578cfa22009c281c780fe)):
+
+```rust,editable
+# extern crate rand;
+
+// import commonly used items from the prelude:
+use rand::prelude::*;
+
+fn main() {
+    // We can use random() immediately. It can produce values of many common types:
+    let x: u8 = random();
+    println!("{}", x);
+
+    if random() { // generates a boolean
+        println!("Heads!");
+    }
+
+    // If we want to be a bit more explicit (and a little more efficient) we can
+    // make a handle to the thread-local generator:
+    let mut rng = thread_rng();
+    if rng.gen() { // random bool
+        let x: f64 = rng.gen(); // random number in range [0, 1)
+        let y = rng.gen_range(-10.0, 10.0);
+        println!("x is: {}", x);
+        println!("y is: {}", y);
+        println!("Number from 0 to 9: {}", rng.gen_range(0, 10));
+    }
+    
+    // We can also interact with iterators and slices:
+    let arrows_iter = "➡⬈⬆⬉⬅⬋⬇⬊".chars();
+    println!("Lets go in this direction: {}", arrows_iter.choose(&mut rng).unwrap());
+    let mut nums = [1, 2, 3, 4, 5];
+    nums.shuffle(&mut rng);
+    println!("I shuffled my {:?}", nums);
```

Comment:
The sentence printed to the screen is a bit weird:
```
I shuffled my [3, 5, 2, 1, 4]
```
Maybe just add "numbers:"?

Reply:
It's not a very good one, I admit. Hint: 'shuffled' is a synonym of 'confused'.

Reply:
Fair enough.